### PR TITLE
adds lasagna

### PR DESCRIPTION
**Pull request by stegglca**
lasagna is good

### CODE REVIEW

**Review by miikyla — DISMISSED** [review]
This is acceptable

File: Foods.yaml [hunk 1]
```diff
@@ -6,4 +6,16 @@
...
+
+# Add your favourite food to the list.
+# Try to maintain valid .yaml formatting.
+# yaml is the markup language we use for config and data files.
+# It's like XML or JSON, but nicer for humans to read.
+# (yes, yaml does have semantic whitespace like python)
+
+FavouriteFoods:
+  # Add the food to this list on this line, if possible.
+  - ["pizza", "kale_smoothies"]
```
Comment (Bidski) [inline Foods.yaml]: This is very wrong
```suggestion

  - [pizza", "kale_smoothies", "cheese", "pie", "lasagna"]
```

**Review by Bidski — CHANGES_REQUESTED** [review]

**Review by moosahassanx — DISMISSED** [review]
I approve that lasagna is good 👍🏿

File: Foods.yaml [hunk 2]
```diff
@@ -6,4 +6,5 @@
 
 FavouriteFoods:
   # Add the food to this list on this line, if possible.
-  - ["pizza", "kale_smoothies", "cheese", "pie", "Carbonara"]
+
+  - ["pizza", "kale_smoothies", "cheese", "pie", "Carbonara", "Lasagna]
```
Comment (KipHamiltons) [inline Foods.yaml:10]: It was pointed out to me yesterday that we should try to stick with `snake_case` for `yaml` files
```suggestion
  - ["pizza", "kale_smoothies", "cheese", "pie", "carbonara", "lasagna]
```

**Pull request closed without merging**